IBPSA/Fluid/FixedResistances/PressureDrop.mo

```diff
@@ -17,6 +17,11 @@ protected
   final parameter Boolean computeFlowResistance=(dp_nominal_pos > Modelica.Constants.eps)
     "Flag to enable/disable computation of flow resistance"
    annotation(Evaluate=true);
+  final parameter Real coeff(unit="")=
```

Comment:
Is there a reason for `coeff` to be public rather than protected?
Is there a reason for settings its units to `""` (and what does this mean? No units, or non-specified units?)

Reply:
`Coeff` is/was protected, so I presume you misread or mistyped?

There is no specific reason for this other than that I copied and modified:
```
final parameter Real k(unit="") = if computeFlowResistance then 
        m_flow_nominal_pos / sqrt(dp_nominal_pos) else 0
    "Flow coefficient, k=m_flow/sqrt(dp), with unit=(kg.m)^(1/2)";
```
I suppose this should be changed too then?

Reply:
fixed in 5bbaea953be10a86fe1db6c928e4fd90237f2cd8

Reply:
You are right, `coeff` was already protected. I will merge it.